omnibus/config/software/openscap.rb

```diff
@@ -39,28 +39,11 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
-  # Fixes since release 1.3.7
-  patch source: "0006-Use-correct-format-specifier.patch", env: env
-  patch source: "0007-Fix-leaked-variable.patch", env: env
-  patch source: "0008-Fix-a-leaked-variable.patch", env: env
-  patch source: "0009-Fix-Wint-conversion-error-building-with-clang.patch", env: env
-  patch source: "0010-Remove-reference-to-PROC_CHECK.patch", env: env
-  patch source: "0015-Fix-leak-of-session-skip_rules.patch", env: env
-  patch source: "0016-Fix-leak-of-dpkginfo_reply_t-fields.patch", env: env
-  patch source: "0018-Fix-leak-of-component_colls-in-_oval_component_evalu.patch", env: env
-  patch source: "0020-Fix-build-on-FreeBSD.patch", env: env
-  patch source: "0036-Fix-leak-of-regex-structure-in-oval_fts-in-error-cas.patch", env: env
-  patch source: "0037-Free-xmlDoc-structure-at-the-end-of-xccdf_session_lo.patch", env: env
-  patch source: "0041-Fix-implicitly-declared-function.patch", env: env
-  patch source: "0042-Plug-a-memory-leak.patch", env: env
-  patch source: "0043-Fix-other-occurences-of-oscap_htable_add.patch", env: env
-
   patch source: "get_results_from_session.patch", env: env # add a function to retrieve results from session
   patch source: "session_result_free.patch", env: env # add a function to free results from session
   patch source: "010_perlpm_install_fix.patch", env: env # fix build of perl bindings
   patch source: "dpkginfo-cacheconfig.patch", env: env # work around incomplete pkgcache path
   patch source: "dpkginfo-init.patch", env: env # fix memory leak of pkgcache in dpkginfo probe
-  patch source: "shadow-chroot.patch", env: env # handle shadow probe in offline mode
```

Comment:
Should this patch also be removed from `omnibus/config/patches/openscap` if it's not used anymore?

Reply:
Indeed. It was an oversight.